deploy/roles/postgresql-server/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
 
 - name: Include variable for os
   include_vars: rhel_7.yml
-  when: "{{ ansible_distribution_major_version }} == 7"
+  when: ansible_distribution_major_version == 7
```

Comment:
This will fail, since the version number is a string; should be `== "7"`.  I'm not going to tell you how much of my life I wasted discovering that.

Reply:
Ah, thanks much!